modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -202,7 +204,7 @@ public static com.liferay.object.model.ObjectField toObjectField(
 		return serviceBuilderObjectField;
 	}
 
-	private static void _addListTypeEntries(
+	private static void _addOrUpdateListTypeEntries(
```

Comment:
My suggestion for the name of this method is `_addOrDeleteListTypeEntries`, what do you think?